bigquery/snippets/snippet.go

```diff
@@ -305,3 +305,56 @@ func exportToGCS(client *bigquery.Client, datasetID, tableID, gcsURI string) err
 	// [END bigquery_export_gcs]
 	return nil
 }
+
+func importJsonExplicitSchema(client *bigquery.Client, datasetID, tableID string) error {
+	ctx := context.Background()
+	// [START bigquery_load_table_gcs_json]
+	gcsRef := bigquery.NewGCSReference("gs://cloud-samples-data/bigquery/us-states/us-states.json")
+	gcsRef.SourceFormat = bigquery.JSON
+	gcsRef.Schema = bigquery.Schema{
+		{Name: "name", Type: bigquery.StringFieldType},
+		{Name: "post_abbr", Type: bigquery.StringFieldType},
+	}
+	loader := client.Dataset(datasetID).Table(tableID).LoaderFrom(gcsRef)
+
+	loader.WriteDisposition = bigquery.WriteEmpty
```

Comment:
nit: move to the previous line. the newline after is fine.

explanation: the WriteDisposition is part creating the loader, so it should belong with the code that created it. It doesn't seem to be part of the call to loader.Run, so shouldn't be next to that piece of code.